src/quo2/components/dividers/strength_divider/view.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.dividers.strength-divider.view
+  (:require [react-native.core :as rn]
+            [quo2.components.dividers.strength-divider.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [react-native.linear-gradient :as linear-gradient]
+            [react-native.svg :as svg]
+            [quo2.components.icon :as icon]))
+
+(def status-list
+  {:very-weak   {:default-text "Very weak"
+                 :level        1
+                 :color        colors/danger-60
+                 :percentage   20}
+   :weak        {:default-text "Weak"
+                 :level        2
+                 :color        (colors/custom-color :orange 60)
+                 :percentage   40}
+   :okay        {:default-text "Okay"
+                 :level        3
+                 :color        (colors/custom-color :yellow 60)
+                 :percentage   60}
+   :strong      {:default-text "Strong"
+                 :level        4
+                 :color        colors/success-60
+                 :percentage   80}
+   :very-strong {:default-text "Very strong"
+                 :level        5
+                 :color        colors/success-60
+                 :percentage   100}
+   :info        {:default-text "Tips to improve your password"
+                 :level        0
+                 :color        colors/white-opa-40}
+   :alert       {:default-text "Alert message"
+                 :level        0
+                 :color        colors/danger-60}})
+
+(def status-indicator-radius 6.5)
+
+(defn view
+  "Options
+   - `status` Status keyword (`:very-weak`/`:weak`/`:okay`/`:strong`/`:very-strong`/`:info`/`:alert`)
+
+   `text` message string(only works when status is `info`/`alert`)"
+  [{:keys [status] :or {status :very-weak}} text]
+  (let [color        ((status-list status) :color)
+        percentage   ((status-list status) :percentage)
+        default-text ((status-list status) :default-text)
+        c            (* 2 Math/PI status-indicator-radius)]
```

Comment:
what is `c`? could you use a more descriptive variable name

Reply:
It's the circumference(2πr) of the progress circle. I will update the variable name.